src/config.ts

```diff
@@ -27,12 +28,15 @@ export const updateCurrentConfig = (siteCfg, _directives) => {
 
   cfg = assignWithDepth(cfg, sumOfDirectives);
 
+  // @ts-ignore
```

Comment:
Over here instead of `@ts-ignore` we can use `sumOfDirectives.theme && sumOfDirectives.theme in theme` and mark the `sumOfDirectives` variable as a `MermaidConfig` when declared on about line 20.

Reply:
Done